CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+Contributing
+============
+
+If you’re not sure this is the correct place to file an issue then please file an issue on the [mozilla/addons] project instead.
+
+Before contributing code, please note:
+
+-   You agree to license your contributions under the [license].
```

Comment:
Ah that's pandoc's fault :) - I converted this from rst here: https://github.com/mozilla/addons-server/blob/master/CONTRIBUTING.rst